Client/Ecosia/Analytics/Analytics.swift

```diff
@@ -135,6 +138,14 @@ final class Analytics {
         track(event)
     }
     
+    func userSearchesViaBingABTest() {
```

Comment:
Is this event required by BI? I don't find it in the ticket spec. 

I am asking because I consider event volume very high and I don't think it's relevant for the experiment as we count searches via the BING portal already.

Reply:
We were adding this analytics in the other experiment so I thought would be still needed as the intrinsic scenario doesn't change. Worth re-assess with Product?

Reply:
Confirmed with David and removed ✅